### ISSUE TITLE
Import dataTables translations with ES6

### ISSUE THREAD

**Issue by eleggua999** [DataTables/Plugins#352]
It could be possible:
1. Get plugins package with nmp:
```
npm install datatables.net-plugins
```
2. Create file, ex. translations.dataTables.js, with:
```
export function initDataTableTranslations() {
    return require('datatables.net-plugins/i18n/German.lang');
}
```
3. And in another file:
```
import { initDataTableTranslations } from './translations.datatables.js'
```
But translation files have  'comments' at the beginning, and JSONs don't have comments, so this doesn't compile and the only option is to get it from cdn or make your own files.

**Comment by DataTables** [comment]
I agree. I've always seen source control as a way to store the source, but it is increasingly obvious that isn't enough and the built files need to be included as well. I will be updating the language files to include build information. At the moment you'd need to get the JSON from the DataTables CDN, or strip out the meta data.

**Comment by rwngallego** [comment]
It would be better if we use JSON standard files instead of .lang, as the `language` option expects a JSON object. It would be easier to load that file using ES6. The metadata could be added into the same JSON file but in a separate object. Something like:
```
{
   "meta": [...],
   "translations": [...]
} 
```
